comparisons/matrix_RE.txt

```diff
@@ -377,6 +377,7 @@ TTbarFS14TeVPU2021wf13434p0 13434.0_TTbar_14TeV+*2021*FS*PU*/step1.root RECO
 TTbar14TeV2023wf12434p0 12434.0_TTbar_14TeV+*2023*/step3.root RECO
 TTbar14TeVPU2023wf12634p0 12634.0_TTbar_14TeV+*2023PU*/step3.root RECO
 TTbar14TeVPU2023nanoEDMwf12634p0 12634.0_TTbar_14TeV+*2023PU*/step5.root NANO
+TTbar14TeVPU2023nanoEDMin126Xwf2500p601 2500.601_mc126X+TTBarMINIAOD*/step2.root NANO
```

Comment:
@slava77 , for my education, `TTbar14TeVPU2023nanoEDMin126Xwf2500p601` is just an arbitrary string which should be unique for ecah entry in this file ... right?

Reply:
it's supposed to be a shorter representation of the full directory name with occasionally an insert/suffix related to the file/process/step in the workflows.
Uhm, 2023 is wrong though, it's a 2022 wf, let me fix that.